### DISCUSSION TITLE
Mailchimp features only for Mailchimp Admins

### DISCUSSION THREAD

**Discussion by ChrisChasm** [DiscipleTools/disciple-tools-mailchimp#4] [Ideas]
Should we limit the mailchimp tile  and fields only to a Mailchimp Admin role? I think showing the tile to all users might be negative for a couple reasons: 
1. For most multipliers it is just clutter to the contact page.
2. You don't want to give power to ever user to subscribe contacts to mailchimp marketing without the email owner giving permission first. The goes against most Spam Laws/GDPR considerations, and Mailchimp's requirements. We need to protect the opt-in process, and giving all uses the ability to opt in people without their consent could have consequences.

So my nomination is to create a role with the plugin called Mailchimp Admin and only show the fields and tiles to that person, who is administrating the mailchimp newsletters.

**Comment by kodinkat** [comment]
Hey @ChrisChasm - Thanks for that and from a regulatory standpoint, understood and agreed! Would there ever be an instance whereby non-MC Admin roles may need access? If so, it might be an idea to also update the admin panels so as to provide the ability to cherry-pick roles of who can access.

If not, then agreed; a new MC Admin role and we just manually assign users accordingly...

Also, another thought which came to mind; if we do decide to hide MC tile; would it be an idea to relocate MC First & Last Names (assuming other roles are allowed to update names)? As currently; these fields are a workaround to the single DT Name field and not having to concatenate F & L names.

**Comment by corsacca** [comment]
Implemented with: https://github.com/DiscipleTools/disciple-tools-mailchimp/commit/2f1d023189aeed99facfd7690adce38ab3a12812
